docs/_code-formatting.md

```diff
@@ -0,0 +1,2 @@
+2-space tabs
+alphabetical css properties & sass/scss nesting
```

Comment:
If we are going to list these out as must haves, we should update the stylelint rules appropriately:

https://stylelint.io/user-guide/rules/list

Reply:
@burnwatt I'm looking into what we might add from Stylelint in the near future.

To clarify, this page is hidden until we decide on the rules. (Any docs page with and underscore at the front is hidden `_example-doc-page-name.md`)